TechnicalDocuments/FastlaneMatch.md

```diff
@@ -0,0 +1,121 @@
+Fastlane Match
+==============
+
+Fastlane Match is a tool for managing certificates and provisioning profiles centrally. It downloads certificates and profiles from a GitHub repository or Google Cloud Storage bucket. For safety centrally stored files are encrypted.
+
+Fastlane match [documentation](https://docs.fastlane.tools/actions/match/).
+
+## Several Targets and Types
+
+Best practise for managing different bundle ids is to create one branch for each of them. Default values for the git URL, branch, app identifier and build type are read from the `Matchfile`.
+
+It can happen that code signing certificates for more than one target or type is needed for a build. This can be handled by making several calls to match.
+
+```ruby
+match(app_identifier: "com.best.app.ever.seriously", type: "adhoc", git_branch: "best_app_ever_ad_hoc_signing")
+match(app_identifier: "com.best.app.ever.seriously", type: "development", git_branch: "best_app_ever_dev_signing")
+```
+
+Information about which repository and branches we use for the Babylon apps can be found in our internal Confluence Wiki.
+
+## Update Contents Manually
+
+Fastlane Match provides a utility for nuking all existing certificates and provisioning profiles and replace them with new ones generated by Fastlane Match. This is, however, not always possible. Apple does not allow more than two distribution certificates per account, so it can happen that Fastlane Match cannot create a new distribution certificate and is unable to figure out which existing certificate to link with the provisioning profile. Nuking existing enterprise certificates can be a really bad idea as in house apps will stop working.
+
+Since the files in the GitHub repository are encrypted one cannot update them directly. Fastlane Match provides a couple of helper functions. To clone the repository, open a ruby console and call `Match::GitHelper.clone`. Babylon specific credentials can be found in our 1Password account.
```

Comment:
Since the files in the GitHub repository are encrypted one**s, we** cannot update them directly.

Reply:
I think the meaning was "Since the files in the GitHub repository are encrypted, one cannot update them directly" 😛 ("One" used here as the indefinite pronoun). but agree a comma was missing nonetheless 😉

Reply:
ah `one` as 'someone' 👍 got it.